src/fosslight_util/parsing_yaml.py

```diff
@@ -46,27 +50,28 @@ def parsing_yml(yaml_file, base_path):
                     oss_list.append(item)
                     license_list.extend(item.license)
                     idx += 1
+    except AttributeError as ex:
+        _logger.error(f"Not supported yaml file format {ex}")
     except yaml.YAMLError:
         _logger.warning(f"Can't parse yaml - skip to parse yaml file: {yaml_file}")
     return oss_list, set(license_list)
 
 
-def find_all_oss_pkg_files(path_to_find, file_to_find):
+def find_sbom_yaml_files(path_to_find):
```

Comment:
Make sure there are no Scanners using `find_all_oss_pkg_files`.

Reply:
It is not used anywhere else. only used in FL Prechecker